tests/datatypes/test_bytes.py

```diff
@@ -474,6 +474,27 @@ def test_istitle(self):
             print(b"py.bee".title())
         """)
 
+    def test_split(self):
+        self.assertCodeExecution("""
+            print(b''.split())
+            print(b'py bee'.split())
+            print(b'pyXbXee'.split(b'X'))
+            print(b'pyXbee'.split(b'z'))
+            print(b'pyZZbee'.split(b'ZZ'))
+            print(b'pybebyp'.split(b'e', 1))
```

Comment:
Could you add a test for the case: `b'one two three'.split(maxsplit=1)`

Reply:
I added the test case you have suggested and the check of errors in the beginning of the method, thank for your advices!